packages/integrations/useAsyncValidator/index.md

```diff
@@ -9,7 +9,7 @@ Wrapper for [`async-validator`](https://github.com/yiminghe/async-validator).
 ## Install
 
 ```bash
-npm i async-validator
+npm i async-validator@^4.2.5
```

Comment:
```suggestion
npm i async-validator@4
```

If that would be better?

Reply:
It's hard to say that other libraries will not have a breaking change in a patch or minor release though they are not supposed to, so maybe more clearly version will be better

Reply:
I have tried, if we execute `npm i async-validator@^4.2.5`, then npm will add a clear field `async-validator: "^4.2.5"` in `package.json`, that's maybe safer

Reply:
> It's hard to say that other libraries will not have a breaking change in a patch or minor release though they are not supposed to, so maybe more clearly version will be better

Well, if that happens, it's their fault and we don't have control over it.

The main thing I am worried about is that we would only update them manually. For example, to say two years later, `async-validator` could go something like `4.99.0` with a lot of security fixes, but ppl come to our docs will install the old `4.2.5`.

Unless we will use a script to update this to the version we tested (from our dev deps) and update automatically when we bump the version, I'd prefer to not set them too strict to have ppl on the latest supported version.

Reply:
Well, that's true. I'll commit some changes to all the docs in `integrations`

Reply:
And I found we should use `npm i async-validator@^4`, or npm will add a specific version `sync-validator: "4"` in `package.json` and I don't think that's what we want